skbio/metadata/_mixin.py

```diff
@@ -311,7 +283,7 @@ def __init__(self, positional_metadata=None):
 
     def _init_(self, positional_metadata=None):
         if positional_metadata is None:
-            self._positional_metadata = None
+            del self.positional_metadata
```

Comment:
Thanks! It looks a little odd but allows the "empty" metadata representation to be defined in a single place (the property deleter).